### DISCUSSION TITLE
Benevnelse i inputfelt

### DISCUSSION THREAD

**Discussion by Stormoen** [fremtind/jokul#2454] [Annet]
Savner et inputfelt med mulighet for benevnelser som "kr", "km" osv.
Tenker et tilsvarende inline oppsett som søk-ikon ville passet utmerket.

Noen fler som har dette behovet?

![Skjermbilde 2021-11-08 kl  15 15 32](https://user-images.githubusercontent.com/64519175/140757611-56534baf-8049-4c07-ab97-cc0046f1dab3.png)
t?

**Comment by Slaine2** [comment]
ja, det mangler i komponentbiblioteket.
Jeg lurte fortsatt på om vi også trenger m2 (boligforsikring/@AudunBerdal ) eller en annen enhet?

**Reply by onordbo** [reply]
Med BSF på trappa vil vel m2 definitivt være relevant, eller hva? 🤠

**Comment by ivnai** [comment]
Oh ja, vi bruker sånn i boligselgerforsikring meglerportal.  Mye bedre er Heidi ditt forslag
![Screenshot 2021-11-08 at 16 02 46](https://user-images.githubusercontent.com/93194579/140766193-4f3290ac-6d1d-4c42-891a-ab63279781ef.png)

**Reply by ivnai** [reply]
hva skjer om man trenger å bruke både, handling og enhet

**Reply by Stormoen** [reply]
Har du eksempel på handling @ivnai ?

**Reply by ivnai** [reply]
Ikke fra real life, prøvde å forestille mulige cases @Stormoen 
![Screenshot 2021-11-15 at 14 09 34](https://user-images.githubusercontent.com/93194579/141787390-7e89ab91-d66c-41a8-97ee-ccc729baf017.png)

**Comment by HavardPede** [comment]
![image](https://user-images.githubusercontent.com/30959515/140768504-364b39ad-9b0b-4f71-92fe-4e4d6d2fe53a.png)
Vi har også sett behovet for dette i SAM.

**Comment by onordbo** [comment]
## Forum update 09/11/21
Teamet går vekk fra prefix der det finnes, og suffix innføres som standard for input-felter med benevnelse. @piofinn ser på utvikling sammen med @Stormoen, som parallellt ferdigstiller designet i en Figma-branch.

**Comment by piofinn** [comment]
# Avstemning! 📝

Hvor vil vi ha benevningen i inputfeltene? Reager med tilhørende emoji for å stemme:

🎉: Inne i feltet, alt er lov **[innhold kr]** / **[innhold x]** / **[innhold kr x]**
👀: Inne i feltet, _enten_ handling _eller_ benevnelse **[innhold x]** / **[innhold kr]**
🚀: Utenfor feltet **[innhold x]** kr

Klikk på alternativet du vil stemme på under

**Reply by piofinn** [reply]
@fremtind/design
